tests/test_perf_analyzer_config_generator.py

```diff
@@ -314,6 +314,31 @@ def test_perf_analyzer_flags(self):
         self._run_and_test_perf_analyzer_config_generator(
             yaml_content, expected_configs)
 
+    def test_multiple_models(self):
+        """
+        Test Multiple Models:  
+            Confirm that if multiple model names are passed in, that
+            they will end up in the PA CLI
+        """
+        args = [
+            'model-analyzer', 'profile', '--model-repository', 'cli_repository',
+            '-f', 'path-to-config-file', '--profile-models', "model1,model2"
+        ]
+        config = self._evaluate_config(args, yaml_content="")
+        pacg = PerfAnalyzerConfigGenerator(
+            config, [model.model_name() for model in config.profile_models],
+            config.profile_models[0].perf_analyzer_flags(),
+            config.profile_models[0].parameters())
+
+        perf_analyzer_configs = []
+        while not pacg.is_done():
+            perf_analyzer_configs.append(pacg.next_config())
+
+        self.assertGreater(len(perf_analyzer_configs), 0)
```

Comment:
while some of these `self.assert<SOMETHING>()` are necessary and/or convenient, i generally think it's more readable to just use `self.assertTrue()` and put the condition inside as regular python

Reply:
fixed